rest_framework_json_api/renderers.py

```diff
@@ -606,3 +607,66 @@ def render(self, data, accepted_media_type=None, renderer_context=None):
         return super(JSONRenderer, self).render(
             render_data, accepted_media_type, renderer_context
         )
+
+
+class BrowsableAPIRenderer(renderers.BrowsableAPIRenderer):
+    template = 'rest_framework_json_api/api.html'
+    includes_template = 'rest_framework_json_api/includes.html'
+
+    def get_context(self, data, accepted_media_type, renderer_context):
+        context = super(BrowsableAPIRenderer, self).get_context(
+            data, accepted_media_type, renderer_context
+        )
+        view = renderer_context['view']
+
+        context['includes_form'] = self.get_includes_form(view)
+
+        return context
+
+    @classmethod
+    def _get_included_serializers(cls, serializer, prefix='', already_seen=None):
+        if not already_seen:
+            already_seen = set()
+
+        if serializer in already_seen:
+            return []
+
+        included_serializers = []
+        already_seen.add(serializer)
+
+        if isinstance(serializer, str):
```

Comment:
`utils.get_included_serializers` handles the imports in a general way and should be used.